### PR TITLE
New inst clients

### PR DESCRIPTION
**Pull request by jreidinger**
Example conversation to new inst clients.
Result is quite good according to codeclimate: https://codeclimate.com/github/yast/yast-bootloader/compare/new_inst_clients

### CODE REVIEW

**Comment by jreidinger** [comment]
I successfully test code on real installation.

File: src/include/bootloader/routines/switcher.rb [hunk 1]
```diff
@@ -73,8 +73,8 @@ def blRead(reread, avoid_reading_device_map)
     # Reset bootloader-specific settings
     # @param [Boolean] init boolean true if basic initialization of system-dependent
```
Comment (mvidner) [inline src/include/bootloader/routines/switcher.rb]: This comment is now useless.

File: src/lib/bootloader/grub2base.rb [hunk 2]
```diff
@@ -79,7 +79,7 @@ def Update
 
     # Reset bootloader settings
     # @param [Boolean] unused
```
Comment (mvidner) [inline src/lib/bootloader/grub2base.rb]: unused -> removed :)

**Comment by jreidinger** [comment]
@mvidner both removed parameter documentation removed.

**Comment by mvidner** [comment]
Thanks, LGTM.

**Pull request merged**